Evergreen/Manifests/IrfanView.json

```diff
@@ -0,0 +1,45 @@
+﻿{
+    "Name": "IrfanView",
+    "Source": "https://www.irfanview.com/",
+    "Get": {
+        "Uri": "https://www.irfanview.com/",
+        "MatchVersion": "<strong>version (\\d+(\\.\\d+){1,4})</strong>"
```

Comment:
The approach with Evergreen is to not scrape HTML. When this page changes, there's a good chance the function will break